.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+# This workflow will upload a Python Package using Twine when a release is created
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#publishing-to-package-registries
```

Comment:
I think this is a stale comment; it's the GH action that uploads the artifacts, not twine.

Also, I think this is the current link https://docs.github.com/en/actions/use-cases-and-examples/building-and-testing/building-and-testing-python#publishing-to-pypi

Reply:
I'm not even sure where that part came from. I was just grabbing a simple template and updating to make sure it worked correctly on a test repo.

Reply:
Those comments aren't needed anyway so I removed them.